### PR TITLE
set retrieveAgentState default to false

### PR DESCRIPTION
**Pull request by fwang-teradici**

### CODE REVIEW

File: Deploy-CAM.ps1 [hunk 1]
```diff
@@ -117,7 +117,7 @@ Param(
     $camManagementUserGroup = $null,
 
     [switch]$updateSPCredential,
-    [bool]$retrieveAgentState = $true,
+    [bool]$retrieveAgentState = $false,
     [bool]$showAgentState = $true
```
Comment (cptera) [inline Deploy-CAM.ps1]: should just make this false too even though it doesn't do anything

Comment (fwang-teradici) [reply Deploy-CAM.ps1]: I think true is better as default so custom only need to set retrieveAgentState is true and it shows the state (normally people like to see if the state is there)

Comment (cptera) [reply Deploy-CAM.ps1]: i think we want to be consistant between v1 and v2

Comment (cptera) [reply Deploy-CAM.ps1]: nevermind, looks liek thats what v2 does

**Pull request merged**